tests/test_install.py

```diff
@@ -39,6 +39,26 @@ def install_package(capsys, pipx_temp_env, caplog, package, package_name=""):
     assert "WARNING" not in caplog.text
 
 
+def install_multiple_packages(capsys, pipx_temp_env, caplog, packages, package_names=()):
```

Comment:
Let's not copy too much, and create a unified `install_packages()` function.

Reply:
Done this, but left the existing tests that use `parametrize` to run the install commands separately, so that there's still some level of testing installing single and multiple packages at once.